pkg/kv/kvserver/kvflowcontrol/kvflowcontrol.go

```diff
@@ -416,12 +416,13 @@ func (t Tokens) String() string {
 
 // SafeFormat implements the redact.SafeFormatter interface.
 func (t Tokens) SafeFormat(p redact.SafePrinter, verb rune) {
-	sign := "+"
 	if t < 0 {
-		sign = "-"
+		p.SafeRune('-')
 		t = -t
+	} else {
+		p.SafeRune('+')
 	}
-	p.Printf("%s%s", sign, humanize.IBytes(uint64(t)))
+	p.SafeString(redact.SafeString(humanize.IBytes(uint64(t))))
```

Comment:
instead of `SafeString`, can we update `humanize.IBytes` so that it returns a `SafeString` right away? this will help avoid similar issues for all other usages of IBytes.

Reply:
We have a `humanizeutil.IBytes` helper returning a `SafeString`, so could use it here. It's slightly different in that it doesn't prepend the string with `+` for positive values, but maybe it's fine to replace. Or we could add a "signed" variant of this helper next to it.

Reply:
ah i see, didn't realize that the `humanize` package is 3rd party. i'd be fine with either approach you suggest.

Reply:
BTW, the format (1st arg) of the `printf` is always considered safe. So, the sign can be added as a part of the format.

```go
func (t Tokens) SafeFormat(p redact.SafePrinter, verb rune) {
	if t < 0 {
		p.Printf("-%s", humanizeutil.IBytes(int64(-t)))
		return
	}

	p.Printf("+%s", humanizeutil.IBytes(int64(t)))
}
```

Reply:
Nice suggestion, done something similar. It wastes one `if t < 0` comparison (since `humanizeutil.IBytes` internally also does it), but it should be fine.